ssd1306/ssd1306.c

```diff
@@ -189,12 +176,26 @@ void ssd1306_UpdateScreen(void) {
     //  * 32px   ==  4 pages
     //  * 64px   ==  8 pages
     //  * 128px  ==  16 pages
-    for(uint8_t i = 0; i < SSD1306_HEIGHT/8; i++) {
-        ssd1306_WriteCommand(0xB0 + i); // Set the current RAM page address.
-        ssd1306_WriteCommand(0x00);
-        ssd1306_WriteCommand(0x10);
-        ssd1306_WriteData(&SSD1306_Buffer[SSD1306_WIDTH*i],SSD1306_WIDTH);
-    }
+#if defined(SSD1306_USE_SPI)
+	ssd1306_WriteCommand(0xB0); // Set the current RAM page address.
```

Comment:
Why did you remove `+i` ?

Reply:
Because the addressing mode was initialized to be Horizontal Addressing Mode in ssd1306_Init().
That is, the address of the first column of the 2nd page is the address of the last column of the 1st page + 1.
So you needn't the for() loop to transfer the fram buffer one page by one page, just transfer all bytes one by one, as if there's only one page and its length is SSD1306_BUFFER_SIZE. I've tested OK.